lxml-stubs/etree.pyi

```diff
@@ -241,6 +241,12 @@ class _ElementTree:
     def iter(
         self, tag: Optional[_TagSelector] = ..., *tags: _TagSelector
     ) -> Iterable[_Element]: ...
+    def parse(
+        self,
+        source: _FileSource,
+        parser: XMLParser | None = ...,
+        base_url: _AnyStr | None = ...,
```

Comment:
I don't think we started using "|" anywhere yet. Let's stick to `Optional[]` for now.

Reply:
Sounds good. The contribution guidelines just pointed to typeshed, and they prefer pipe. `Optional[]` works for me.